### PR TITLE
fix: Technical SEO improvements

### PR DESCRIPTION
**Pull request by renukote**
## Description 

Provide a concise summary of the changes made in this pull request
- Adds 10 new redirects

## Pull request type

Check the appropriate box:

- [ ] Review Fixes
- [ ] Documentation Overhaul
- [ ] Feature/Story
    - Link one or more Engineering Tickets
        * 
- [ ] A-Force
- [ ] Error in documentation
- [X] Maintenance

## Documentation tickets

 Link to one or more documentation tickets:
 - 

## Checklist

From the below options, select the ones that are applicable:

- [ ] Checked for Grammarly suggestions.
- [ ] Adhered to the writing checklist.
- [ ] Adhered to the media checklist.
- [ ] Verified and updated cross-references or added redirect rules.
- [ ] Tested the redirect rules on deploy preview.
- [ ] Validated the modifications made to the content on the deploy preview.
- [ ] Validated the CSS modifications on different screen sizes.

### CODE REVIEW

File: website/vercel.json [hunk 1]
```diff
@@ -408,7 +408,8 @@
     },
     {
       "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
-      "destination": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud"
+      "destination": "/connect-data/reference/querying-mongodb",
```
Comment (jnikhila) [inline website/vercel.json:411]: The redirect rules should be added as new. These redirect rules are needed as is.

File: website/vercel.json [hunk 2]
```diff
@@ -1201,6 +1202,56 @@
...
+      "destination": "/reference/appsmith-framework/query-object",
+      "permanent": true
+    },
+    {
+      "source": "/write-code/reference/core-concepts/writing-code/ext-libraries",
+      "destination": "/core-concepts/writing-code/ext-libraries",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud",
```
Comment (jnikhila) [inline website/vercel.json:1252]: duplicate redirect

File: website/vercel.json [hunk 3]
```diff
@@ -1201,6 +1202,56 @@
...
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
+      "destination": "/connect-data/reference/querying-mongodb",
```
Comment (jnikhila) [inline website/vercel.json:1233]: this should point to the whitelisting external link

File: website/vercel.json [hunk 4]
```diff
@@ -1201,6 +1202,56 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/reference/appsmith-framework/reference/appsmith-framework/widget-actions/post-message",
```
Comment (jnikhila) [inline website/vercel.json:1207]: This is a wrong URL why do we need to create a redirect for this.

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/vercel.json [hunk 5]
```diff
@@ -1201,6 +1202,56 @@
...
+      "destination": "/reference/appsmith-framework/widget-actions/post-message",
+      "permanent": true
+    },
+    {
+      "source": "/build-apps/how-to-guides/reference/appsmith-framework/widget-actions/window-message-listener",
```
Comment (jnikhila) [inline website/vercel.json:1212]: The same is true for this URL. This is the wrong URL that got indexed.

File: website/vercel.json [hunk 6]
```diff
@@ -1201,6 +1202,56 @@
...
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
+      "destination": "/connect-data/reference/querying-mongodb",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/radio-buttons",
```
Comment (jnikhila) [inline website/vercel.json:1237]: This is the wrong URL why do we need a redirect? This should be removed from the index.

File: website/vercel.json [hunk 7]
```diff
@@ -1201,6 +1202,56 @@
...
+      "destination": "/reference/widgets/radio-group",
+      "permanent": true
+    },
+    {
+      "source": "/reference/readme-1/query-object",
```
Comment (jnikhila) [inline website/vercel.json:1242]: Same as above

Comment (jnikhila) [reply website/vercel.json:1242]: `/reference/readme-1/query-object` - Redirect not needed, should be removed.

File: website/vercel.json (hunk 2 above)
Comment (jnikhila) [reply website/vercel.json:1252]: This needs to be removed as this is a duplicate Redirect.

Comment (jnikhila) [reply website/vercel.json:1252]: And it should point to the external whitelisting link

File: website/vercel.json (hunk 4 above)
Comment (jnikhila) [reply website/vercel.json:1207]: This should be removed too.

File: website/vercel.json (hunk 5 above)
Comment (jnikhila) [reply website/vercel.json:1212]: This should be removed too.

File: website/vercel.json (hunk 3 above)
Comment (Renu-codewalnut) [reply website/vercel.json]: Changed the landing page URL to whitelisting external link

File: website/vercel.json [hunk 8]
```diff
@@ -1201,6 +1201,36 @@
...
+      "destination": "/reference/widgets/maps",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
```
Comment (jnikhila) [inline website/vercel.json:1221]: reference/datasources - Is. a wrong URL. Let's remove it from the index.

Comment (jnikhila) [reply website/vercel.json:1221]: `/reference/datasources/querying-mongodb/whitelist-mongodb-cloud` should be redirected to - https://studio3t.com/knowledge-base/articles/mongodb-atlas-login-ip-whitelisting/#how-to-whitelist-ip-addresses-on-mongodb-atlas

Comment (jnikhila) [reply website/vercel.json:1221]: Only one redirect rule is to be added here for the whitelist.

File: website/vercel.json [hunk 9]
```diff
@@ -1201,6 +1201,36 @@
...
+      "destination": " /connect-data/reference/querying-mongodb/whitelist-mongodb-cloud",
+      "permanent": true
+    },
+    {
+      "source": "/write-code/reference/core-concepts/writing-code/ext-libraries",
+      "destination": "/core-concepts/writing-code/ext-libraries",
+      "permanent": true
+    },
+    {
+      "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
```
Comment (jnikhila) [inline website/vercel.json:1231]: Not needed. As we already have a rule:
```
      "source": "/datasource-reference/querying-mongodb/whitelist-mongodb-cloud",
      "destination": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud"
```
The above rule will handle the redirect and the whitelist rule will take users to the whitelist external link.

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/vercel.json (hunk 8 above)
Comment (Renu-codewalnut) [reply website/vercel.json:1221]: updated

**Comment by jnikhila** [comment]
@Renu-codewalnut - Could you please sign and accept the CLA? It's mandatory for contribution.

File: website/vercel.json (hunk 9 above)
Comment (Renu-codewalnut) [reply website/vercel.json]: removed

File: website/docs/getting-started/setup/instance-configuration/README.mdx [hunk 10]
```diff
@@ -21,14 +21,13 @@ The Admin Settings page allows [superusers](#add-super-admins) on self-hosted in
 If you don’t see the “Admin Settings” icon on the top-right corner, you’re not a super admin for the Appsmith instance and must be granted admin privileges by an existing super admin.
 :::
 
- <ZoomImage
-        src="/img/admin-settings-topbar.png"
-        alt="Admin Settings option is available in the left sidebar"
-        caption="Access admin settings from the top-right corner of the homepage."
-        lazyLoad="true"
+<ZoomImage
```
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/README.mdx:24]: Why is this change needed?

File: website/vercel.json [hunk 11]
```diff
@@ -1201,6 +1201,46 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/widget-reference/file-picker",
```
Comment (jnikhila) [inline website/vercel.json:1206]: This is a wrong URL, can we please add it for deindexing.

File: website/vercel.json [hunk 12]
```diff
@@ -1201,6 +1201,46 @@
...
+      "destination": "/reference/widgets/filepicker",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/map",
```
Comment (jnikhila) [inline website/vercel.json:1211]: Same as above.

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/docs/getting-started/setup/instance-configuration/README.mdx (hunk 10 above)
Comment (renukote) [reply website/docs/getting-started/setup/instance-configuration/README.mdx:24]: sorry, my fomatter changed it when I auto saved it, I'll remove them

File: website/vercel.json (hunk 11 above)
Comment (renukote) [reply website/vercel.json:1206]: deindexed

File: website/vercel.json (hunk 12 above)
Comment (renukote) [reply website/vercel.json:1211]: deindexed

File: website/vercel.json [hunk 13]
```diff
@@ -1205,7 +1205,46 @@
...
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud",
+      "destination": "https://studio3t.com/knowledge-base/articles/mongodb-atlas-login-ip-whitelisting/#how-to-whitelist-ip-addresses-on-mongodb-atlas",
+      "permanent": true
+    },
+    {
+      "source": "/write-code/reference/core-concepts/writing-code/ext-libraries",
```
Comment (jnikhila) [inline website/vercel.json:1230]: This is a wrong url. Let's deindex it, unless we really need a redirect for this.

Comment (renukote) [reply website/vercel.json]: Removed and deindexed

**Review by jnikhila — APPROVED** [review]
